app/views/devise/passwords/edit.html.erb

```diff
@@ -0,0 +1,25 @@
+<h2>Change your password</h2>
```

Comment:
Are we gonna use this? Is there a link to it?

Reply:
There is no link for it,  but should we leave it for later use to changed a users password or is this out of scope